sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -102,10 +102,9 @@ def __init__(
     # Progress report generation should not do IO or wait on other resources.
     #  Without wait, having multiple threads will not improve performance and
     #  will only add complexity.
-    self._progress_thread_pool = futures.ThreadPoolExecutor(max_workers=1)
+    self._progress_thread_pool = UnboundedThreadPoolExecutor()
```

Comment:
Update the comment above? (`one thread is enough ...`)

Reply:
removed.